src/common/util/include/openvino/util/os.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#if defined(_WIN32)
+#    include <windows.h>
+#endif
+
+namespace ov {
+namespace util {
+
+#if defined(_WIN32)
```

Comment:
Define it in .cpp file only it looks like is required only for `_WIN32_WINNT >= 0x0602`.

Reply:
The fn was removed at all